packages/playground/website/src/lib/make-blueprint.tsx

```diff
@@ -17,11 +17,16 @@ export function makeBlueprint(options: MakeBlueprintOptions): Blueprint {
 			wp: options.wp as any,
 		},
 		steps: [
-			{
-				step: 'login',
-				username: 'admin',
-				password: 'password',
-			},
+			options.import
+				? {
+						step: 'importFile',
+						file: options.import,
```

Comment:
do we want to make sure this only includes `http:` or `https:` schemas? I'm not sure if the browser would end up trying to download a `file://` link. hopefully not, but I don't want to open up the possibility of having a playground script lift a file from peoples' systems and then exporting it across the net.

Reply:
Good idea, it makes sense to restrict the kind of value that the GET parameter passes. I'll add a step that ensures the protocol `http` and `https`.

That reminds me, if I understand correctly, this zip file must be served with CORS enabled - similar to plugins and themes, which require a proxy to get around CORS. In which case, it would be a good to document this restriction, since I imagine people will run into the issue, for example, linking to a zip package from a WordPress site's upload folder.

Reply:
Done: [Ensure protocol http or https for import file URL](https://github.com/WordPress/wordpress-playground/pull/610/commits/a999e378276a9a3d565679f78f12d2e97f6c713e).